src/vmm/src/builder.rs

```diff
@@ -217,7 +217,11 @@ pub fn build_microvm_for_boot(
         .map(|vcpu| vcpu.copy_kvm_vcpu_fd(vmm.vm()))
         .collect::<Result<Vec<_>, _>>()?;
 
-    vmm.device_manager.enable_pci()?;
+    if vm_resources.pci_enabled {
+        vmm.device_manager.enable_pci()?;
+    } else {
+        boot_cmdline.insert("pci", "off")?;
```

Comment:
nit: will need to remove all references to `pci=off` from documentation (https://github.com/search?q=repo%3Afirecracker-microvm%2Ffirecracker%20pci%3Doff&type=code)